### PR TITLE
ImmutableOpenMap removal compile fix

### PR DESCRIPTION
**Pull request by petardz**
*Issue #, if available:*

*Description of changes:*

*CheckList:*
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/index-management/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-bin.zip
```
Comment (xluo-aws) [inline gradle/wrapper/gradle-wrapper.properties:3]: There is another PR https://github.com/opensearch-project/notifications/pull/663 to bump the gradle version to 8.1.1

Comment (petardz) [reply gradle/wrapper/gradle-wrapper.properties:3]: 8.1.1 didn't work for me. I got licence file issues and few more

Comment (petardz) [reply gradle/wrapper/gradle-wrapper.properties]: Ah nevermind.. something changed again in meanwhile. 7.6.1 is not working anymore

**Comment by yuye-aws** [comment]
Hi, DCO shows that the sign-off is missing in two of your commits. Please follow the instructions to add your sign-off, thanks!

**Comment by yuye-aws** [comment]
It is good to see you making new commits, but you need to add sign-off to each of your commit. For more details, you can refer to instructions listed in DCO tests.

**Comment by yuye-aws** [comment]
Great, your commit has passed the DCO test! I am here to notifying you of some recent changes on the the main repository. A pull request has just been approved, which includes some fixes with ImmutableOpenMap removal issue. I assume there is some overlapping with your code changes. You can take a look at the latest commit for detailed information.

**Comment by petardz** [comment]
> It is good to see you making new commits, but you need to add sign-off to each of your commit. For more details, you can refer to instructions listed in DCO tests.

oops.. fixed it!

yea I saw @bowenlan-amzn PR

**Pull request closed without merging**